### ISSUE TITLE
Modeling Deep Temporal Dependencies with Recurrent “Grammar Cells”

### ISSUE THREAD

**Issue by guanfuchen** [guanfuchen/videopred#7]
related paper

|摘要|
|---|
|We propose modeling time series by representing the transformations that take a frame at time t to a frame at time t+1. To this end we show how a bi-linear model of transformations, such as a gated autoencoder, can be turned into a recurrent network, by training it to predict future frames from the current one and the inferred transformation using backprop-through-time. We also show how stacking multiple layers of gating units in a recurrent pyramid makes it possible to represent the ”syntax” of complicated time series, and that it can outperform standard recurrent neural networks in terms of prediction accuracy on a variety of tasks.|

![image](https://user-images.githubusercontent.com/22321977/49910949-9235ad00-febf-11e8-852c-4f74ac44b9d6.png)
